internal/live/live.go

```
@@ -0,0 +1,560 @@
+	defaultReconnectMax    = 3
+	defaultHeartbeat       = 10 * time.Second
+	reconnectBackoff       = 200 * time.Millisecond
+	controlStatusStreamEnd = 3
+)
+
+// Config wires one live-monitor session. HTTP and Registry are required;
+// Signer must be set (fail-closed — no silent unsigned dialing).
+type Config struct {
+	HTTP              *httpclient.Client // room page fetches (UA/retry)
+	Registry          *contracts.Registry
+	methods           map[string]string // protoName -> event key (contract protocol_methods)
```

golangci-lint (2.12.2)

[error] field methods is unused

(unused)